src/Symfony/Component/Serializer/Normalizer/AbstractObjectNormalizer.php

```diff
@@ -102,6 +102,16 @@ abstract class AbstractObjectNormalizer extends AbstractNormalizer
      */
     public const PRESERVE_EMPTY_OBJECTS = 'preserve_empty_objects';
 
+    /**
+     * Flag to enable accessing nested atttibutes with the SerializedName annotation.
+     */
+    public const FLATTEN_NESTED_ATTRIBUTES = 'flatten_nested_attributes';
+
+    /**
+     * Spacer used to separate attributes.
+     */
+    protected const FLATTENER = '.';
```

Comment:
This should be `private` (as we are in the process of making the abstract classes anyway) or `public` if it's useful for outside code.